packages/trace-viewer/index.html

```diff
@@ -26,5 +26,12 @@
   <body>
     <div id="root"></div>
     <script type="module" src="/src/index.tsx"></script>
+    <dialog id="fallback-error">
```

Comment:
Why not move it into the index.tsx or below?

Reply:
If it's in `index.tsx` it would never run. `index.tsx` is bundled as an external file (`main.[hash].js`) which is blocked when opening the trace viewer app itself with the `file://` protocol:

<img width="886" alt="Screen Shot 2022-08-08 at 4 25 39 PM" src="https://user-images.githubusercontent.com/11915034/183531384-85258d77-2c42-4799-8e59-f741a3f40f28.png">

The Trace Viewer does not inline all assets unlike the HTML Report which—except for the traceviewer—inlines its bundle into the file instead of having it be external.